core-primitives/sgx-runtime-primitives/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "itp-sgx-primitives"
+version = "0.9.0"
+authors = ["Integritee AG <hello@integritee.network>"]
+edition = "2021"
+
+[dependencies]
+
+# Substrate dependencies
+sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.28" }
+sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.28" }
```

Comment:
Could you add an `std` feature to this toml? Just like here: https://github.com/integritee-network/worker/blob/master/core-primitives/storage/Cargo.toml#L33-L44

Reply:
Thank you, added it in 045e39048a0a98b9ecd76af4f4ec5379b012f8e5.